### DISCUSSION TITLE
Java

### DISCUSSION THREAD

**Discussion by Rystars** [AndroidIDEOfficial/AndroidIDE#519] [Q&A]
how to fix this?
![Screenshot_2022-10-11-07-48-48-131_com itsaky androidide](https://user-images.githubusercontent.com/96302690/194974010-58132178-40c1-433d-9395-7c3345e1b20d.jpg)

**Comment by itsaky** [comment]
This was fixed in recent commits. You can wait for the next release or use the APK from GitHub actions. See #441.
